src/lua/CommunityBalanceMod/Combined/TechData.lua

```diff
@@ -40,6 +40,178 @@ local function GetTechToAdd()
             [kTechDataTooltipInfo] = "EXOSUIT_TECH_TOOLTIP", -- "Allows Exosuits to be purchased"   text for desciption
             [kTechDataResearchName] = "RESEARCH_EXOSUITS_TITLE" -- "Exosuits"               text for left side
         },
+
+        {
+            [kTechDataId] = kTechId.FortressCrag,
+            [kTechDataBioMass] = kCragBiomass,
+            [kTechDataSupply] = kCragSupply,
+            [kTechDataGhostModelClass] = "AlienGhostModel", 
+            [kTechDataMapName] = FortressCrag.kMapName,  -- TODO change to fortress once added
```

Comment:
I assume this comment is referring to the FortressCrag class? If so can we remove the comment?

Reply:
Comments were outdated